pkg/gen/dotnet-templates/kind.cs.mustache

```diff
@@ -0,0 +1,68 @@
+// *** WARNING: this file was generated by the Pulumi Kubernetes codegen tool. ***
+// *** Do not edit by hand unless you're certain you know what you are doing! ***
+
+using System.Collections.Immutable;
+using System.Threading.Tasks;
+using Pulumi.Serialization;
+
+namespace Pulumi.Kubernetes.{{Group}}.{{Version}} {
+    /// <summary>
+    /// {{{Comment}}}{{{PulumiComment}}}
+    /// </summary>
+    public partial class {{Kind}} : Pulumi.CustomResource {
+        {{#Properties}}
+        {{{Comment}}}
+        [Output("{{Name}}")]
+        public {{{PropType}}} {{LanguageName}} { get; private set; } = null!;
+
+        {{/Properties}}
+
+        /// <summary>
+        /// Create a {{Kind}} resource with the given unique name, arguments, and options.
+        /// </summary>
+        ///
```

Comment:
Nit: no need to add an empty line between summary and params, here and in other places